### PR TITLE
Concf 146

### PR DESCRIPTION
**Pull request by bognix**
Remove MyVideo from list of supported providers

~~Don't merge before translations are updated and MyVideo is removed from the list of supported providers~~

**Update**
Translations are uploaded. Branch can be merged

### CODE REVIEW

**Comment by bognix** [comment]
ping @vforge @hakubo @garthwebb

File: extensions/wikia/VideoHandlers/VideoHandlers.i18n.php [hunk 1]
```diff
@@ -50,6 +50,7 @@
 	'videos-error-admin-only' => 'Sorry, only admins of this wiki are permitted to add videos',
 	'videos-initial-upload-edit-summary' => 'created video',
 	'videos-update-edit-summary' => 'updated video',
+	'videos-provider-not-supported' => 'This video provider is not supported. View our list of supported providers [http://community.wikia.com/wiki/Help:Video_Embed_Tool#Supported_sites here].'
```
Comment (hakubo) [inline extensions/wikia/VideoHandlers/VideoHandlers.i18n.php]: This is against accessibility. This should go something like this
```
This video provider is not supported. View our list of [http://community.wikia.com/wiki/Help:Video_Embed_Tool#Supported_sites supported providers ].
```

File: extensions/wikia/VideoHandlers/VideoHandlers.i18n.php [hunk 2]
```diff
@@ -83,7 +84,8 @@
 	'videos-error-old-type-video' => 'Error message when user tries to add a video of a type that is no longer supported.',
 	'videos-error-while-loading' => 'Error message when failing to add a video.',
 	'videos-error-admin-only' => 'Error message that shows up when the wgAllVideosAdminOnly is set to true and a non-admin attempts to upload a video',
-	'videos-update-edit-summary' => 'Edit summary used when updating a video in an article'
+	'videos-update-edit-summary' => 'Edit summary used when updating a video in an article',
+	'videos-provider-not-supported' => 'Message when video provider is not supported'
```
Comment (Warkot) [inline extensions/wikia/VideoHandlers/VideoHandlers.i18n.php]: Maybe `videos-error-provider-not-supported`?

File: extensions/wikia/VideoHandlers/VideoHandlerHelper.class.php [hunk 3]
```diff
@@ -493,4 +493,23 @@ public function updateThumbnailData( $file ) {
...
+	/**
+	 * Check if video's provider is supported base on its URL.
+	 * Logic is partially ported from ApiWrapperFactory::getApiWrapper method.
+	 * @param $url
+	 * @return bool
+	 */
+	public function isVideoProviderSupported( $url ) {
+		global $wgVideoMigrationProviderMap;
+
+		$parsed = parse_url( strtolower( $url ) );
```
Comment (Warkot) [inline extensions/wikia/VideoHandlers/VideoHandlerHelper.class.php]: As you are using only one element of the array later on, you might go for `parse_url( strtolower( $url ), PHP_URL_HOST)` to grab just the host as a string (ref. http://php.net/manual/en/function.parse-url.php).

**Comment by vforge** [comment]
LGTM

**Comment by Warkot** [comment]
LGTM :octocat:

**Pull request merged**